src/structures/Message.js

```diff
@@ -133,6 +134,17 @@ class Message extends Base {
       }
     }
 
+    /**
+     * A collection of stickers in the message
+     * @type {Collection<Snowflake, Sticker>}
+     */
+    this.stickers = new Collection();
```

Comment:
This property is missing in Message's types.

Reply:
Oh I will fix that soon

Reply:
I have committed this :)